### ISSUE TITLE
`dims` of `flip()` without `list` or `tuple` works with a tensor agaist the doc

### ISSUE THREAD

**Issue by hyperkai** [pytorch/pytorch#130641]
### 📚 The doc issue

[The doc](https://pytorch.org/docs/stable/generated/torch.flip.html) of `flip()` says the type of `dims` is `list` or `tuple` as shown below:

> - dims (a list or [tuple](https://docs.python.org/3/library/stdtypes.html#tuple)) – axis to flip on

But, `dims` without `list` or `tuple` works with a tensor against the doc as shown below:

```python
import torch

my_tensor = torch.tensor([[[2, 7, 4], [8, 3, 2]], # 3D tensor
                          [[5, 0, 8], [3, 6, 1]]])
my_tensor.flip(0, 1, 2)
# tensor([[[1, 6, 3],
#          [8, 0, 5]],
#         [[2, 3, 8],
#          [4, 7, 2]]])
```
